homeassistant/components/fritz/common.py

```diff
@@ -257,6 +264,15 @@ def signal_device_update(self) -> str:
         """Event specific per FRITZ!Box entry to signal updates in devices."""
         return f"{DOMAIN}-device-update-{self._unique_id}"
 
+    def _set_guest_wifi_password(self, password: str, length: int) -> None:
```

Comment:
btw. this method seems not to be used any where in the code? 🤔

Reply:
My fault, implemented it in the most recent change. But if we could do the schema validation, it's not necessary and can be removed.